frontend/web/components/modals/CreateSegment.tsx

```diff
@@ -442,7 +442,7 @@ const CreateSegment: FC<CreateSegmentType> = ({
               </Row>
             }
           >
-            <div className='my-4'>
+            <div className='my-4 col-lg-8'>
```

Comment:
what do you think about centering the form? a bit weird not full width on the left no ?

<img width="1207" height="685" alt="image" src="https://github.com/user-attachments/assets/8982a06f-4330-4f93-8b7e-2935f4af183c" />